Example_DynamicFramework/Demo/AppDelegate.m

```diff
@@ -76,6 +76,16 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     //    StreetHawk.notificationTypes = UIRemoteNotificationTypeAlert;
     //    StreetHawk.isDefaultLocationServiceEnabled = NO;
     [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(installRegisterSuccessHandler:) name:SHInstallRegistrationSuccessNotification object:nil];
+    
+    SEGAnalyticsConfiguration *configuration = [SEGAnalyticsConfiguration configurationWithWriteKey:@"Z3KDihiA42uXBPr82xULjKA6Fb0p0v38"];
+    configuration.trackApplicationLifecycleEvents = YES; // Enable this to record certain application events automatically!
+    configuration.recordScreenViews = YES; // Enable this to record screen views automatically!
+    [SEGAnalytics setupWithConfiguration:configuration];
+    
+    
+    StreetHawk.autoIntegrateAppDelegate = YES;
+    [StreetHawk registerInstallForApp:appKey segmentID:[[SEGAnalytics sharedAnalytics] getAnonymousId] withDebugMode:YES];
```

Comment:
Does getAnonymousId change in each call?

Reply:
segmentId  should be just a string, anonymousid may or may not change, better to handle both cases.

But you need to make sure that you get anonymous id after segment integration has been initialized.

https://segment.com/docs/sources/mobile/ios/#how-do-i-know-when-a-destination-is-initialized-

you have to initialize pointzi in the callback to make sure that you get a segment anonymousId when pointzi starts